sway/commands/output/color_profile.c

```diff
@@ -108,7 +116,7 @@ struct cmd_results *output_cmd_color_profile(int argc, char **argv) {
 
 		wlr_color_transform_unref(config->handler_context.output_config->color_transform);
 		config->handler_context.output_config->color_transform = tmp;
-		config->handler_context.output_config->set_color_transform = true;
+		config->handler_context.output_config->color_profile = COLOR_PROFILE_TRANSFORM;
```

Comment:
We should probably error out here if `new_mode != COLOR_PROFILE_TRANSFORM`? We don't want users to be able to use `color_profile --device-primaries icc`.